src/components/tutorials-sidebar/index.tsx

```diff
@@ -15,6 +16,8 @@ import {
 	TutorialSidebarProps,
 } from './types'
 import s from './tutorials-sidebar.module.css'
+import { useRouter } from 'next/router'
+import { ProductSlug } from 'types/products'
```

Comment:
⛏️ can we move these imports up to keep the order of external --> local

Reply:
👍 I dig it, I lean on VSCode auto-importing and forget how messy it ends up 😅 Re-ordered these! ✅